pkg/rancher-desktop/main/snapshots/snapshots.ts

```diff
@@ -0,0 +1,89 @@
+import { spawnFile } from '@pkg/utils/childProcess';
+import Logging from '@pkg/utils/logging';
+import { getRdctlPath } from '@pkg/utils/paths';
+
+import { Snapshot, SpawnResult } from './types';
+
+const console = Logging.snapshots;
+
+function parseFields(line: string): string[] {
+  return line?.split(/\s{2,}/) || [];
```

Comment:
This, and others, will be changed once we have #5569 — Francesco is already aware.

For that reason I'll skim over the `parse*` functions.